lms/djangoapps/grades/models.py

```diff
@@ -251,17 +251,23 @@ def save_grade(cls, **kwargs):
         usage_key = kwargs.pop('usage_key')
         try:
             with transaction.atomic():
-                grade, is_created = cls.objects.get_or_create(
+                grade, is_created = cls.objects.update_or_create(
                     user_id=user_id,
                     course_id=usage_key.course_key,
                     usage_key=usage_key,
                     defaults=kwargs,
                 )
+            log.info(u"Persistent Grades: Grade model saved: {0}".format(grade))
         except IntegrityError:
-            cls.update_grade(user_id=user_id, usage_key=usage_key, **kwargs)
-        else:
-            if not is_created:
-                grade.update(**kwargs)
+            # TODO: enqueue a celery task to update the grade per TNL-XXXX
```

Comment:
https://openedx.atlassian.net/browse/TNL-5471

I'll put the number on this line.